### DISCUSSION TITLE
lantern闪退无法使用

### DISCUSSION THREAD

**Discussion by PiratesFirst** [getlantern/lantern-forum-cn#285] [Q&A]
pc版本今天lantern无缘无故闪退，删除软件和用户数据后，重新下载github最新版本的，同样会闪退。

**Comment by i0qp0i** [comment]
遇到同样问题，莫名其妙就不能用了，双击后在任务栏闪了一个lantern的灰色图标后，就自动消失了，重装软件和重装系统后都没用。

**Reply by PiratesFirst** [reply]
就是这样，设置网页也不弹出来

**Reply by i0qp0i** [reply]
我还以为是我自己的问题呢，原来是很多人都出现了，我当时正在安装stable diffusion的组件，感觉和那个相关，因为我的笔记本还能用lantern。

**Reply by PiratesFirst** [reply]
我的昨天都能用   今天就开始闪退了。   应该时程序更新了什么

**Comment by MaxMKMax** [comment]
@PiratesFirst @i0qp0i 好消息，我们已经修复了 7.8.2 中的崩溃问题，版本 7.8.3 现已推出 https://github.com/getlantern/lantern

**Reply by Zrkfire** [reply]
Not work

**Reply by Naoki19880107** [reply]
not same problem,i likely

**Reply by Naoki19880107** [reply]
not same problem,i likely

**Reply by OceanSJY** [reply]
I met the same issue even if I installed the latest version v7.8.3

**Reply by OceanSJY** [reply]
You mentioned there is a new version v7.8.3, however on release page, it showed the latest version is still v7.8.1?
